src/app/clusters/general-commissioning-server/general-commissioning-server.cpp

```diff
@@ -252,5 +256,6 @@ bool emberAfGeneralCommissioningClusterSetRegulatoryConfigCallback(app::CommandH
 
 void MatterGeneralCommissioningPluginServerInitCallback()
 {
+    DeviceLayer::DeviceControlServer::DeviceControlSvr().SetBreadcrumb(0);
```

Comment:
It should already be getting initialized to 0 in the attribute store...

Reply:
See above comment - should be addressed in separate PR.